recipe/meta.yaml

```
@@ -27,38 +28,57 @@ outputs:
         - python
         - packaging
         - pandas >=1.4.0
+        - shapely >=2.0.0
+        - numpy >=1.22
+        - pyogrio >=0.7.2
         - pyproj >=3.3.0
-        - shapely >=1.8.0
-        - fiona >=1.8.21
+      run_constrained:
+        # geodatabase access
+        - psycopg2-binary>=2.8.0
```

```suggestion
        - psycopg2 >=2.8.0
```
Technically, our package is `psycopg2` **not** `psycopg2-binary` (this one is available on PyPI only)

---

I see that `psycopg2` might get used (largely in the tests) but where does `2.8.0` come from?

---

https://github.com/geopandas/geopandas/blob/v1.0.1/environment-dev.yml#L35
```suggestion
        # https://github.com/geopandas/geopandas/blob/v1.0.1/environment-dev.yml#L35
        - psycopg2 >=2.8.0
```